src/report_table.js

```diff
@@ -631,6 +644,37 @@ looker.plugins.visualizations.add({
       );
     }
 
+    // Here goes the sorting then the vis is built again
+    // If order = true => ascending
+    // If order = false => descending
+    const updateSorting = (order, key, pivot) => {
```

Comment:
I've not deeply looked at this but I wanted to understand if this is following the traditional looker behavior that the sort is done on the client if the results are less than the limit but if the results are more than the limit then the sort has to re run the query first?